Dockerfile-tests

```diff
@@ -0,0 +1,46 @@
+FROM centos:centos6
+MAINTAINER Eric Thompson <erict@puppet.com>
+
+ENV project rototiller
+
+RUN yum -y update && yum -y install \
+  gcc \
+  gcc-c++ \
+  git \
+  make \
+  ruby \
+  ruby-devel \
+  zlib-devel \
+  openssh-server \
+  which \
+  && yum clean all
+
+# install RVM, Ruby, and Bundler
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+RUN \curl -L https://get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm install 2.4"
```

Comment:
I assume you are doing the bash to get the rvm.  Couldn't you just run the source command that it added to your rc?  That would avoid spawning several additional shells which would make it a little faster I think.

Reply:
i'm not sure why, but if you don't say /bin/bash here, it can't find rvm. 
are you saying we can just source the rc file it included?
`source $HOME/.rvm/scripts/rvm`

Reply:
hmmm, that script doesn't exist.